### ISSUE TITLE
Dashboards and controls

### ISSUE THREAD

**Issue by juanramb** [GoogleWebComponents/google-chart#79]
Hi, is there a way of composing a chart with a chartRangeFilter using polymer elements? There is https://developers.google.com/chart/interactive/docs/gallery/controls but don't know if it can be used with google-chart element

**Comment by ebidel** [comment]
The google-chart elements don't support that. Think we'd probably have to create something new like google-chart-dashboard.

**Comment by wesalvaro** [comment]
Yeah, that would be the right approach. I'm also very interested in this, so I'll be taking a look at it.

**Comment by wesalvaro** [comment]
My current idea is to move the data into a behavior and create two elements:

    <google-chart-dashboard>
    <google-chart-control>

Theses elements should be used like so:

    <google-chart-dashboard data="...">
      <google-chart-control type="number-range" for="myChart"></google-chart-control>
      <google-chart id="myChart"></google-chart>
    </google-chart-dashboard>

**Comment by wesalvaro** [comment]
I have a preliminary CL with this functionality in #91.
